src/circuits/atomicSig.ts

```diff
@@ -66,34 +67,14 @@ export class AtomicQuerySigPubSignals
     return await checkQueryRequest(query, outs, schemaLoader);
   }
   async verifyStates(resolver: IStateResolver): Promise<void> {
-    const userStateResolved: ResolvedState = await resolver.resolve(
-      this.userId.bigInt(),
-      this.userState,
-    );
-    if (!userStateResolved.latest) {
-      throw new Error(`only latest states are supported`);
-    }
+    await checkUserState(resolver, this.userId, this.userState);
 
-    const issuerStateResolved: ResolvedState = await resolver.resolve(
-      this.issuerId.bigInt(),
-      this.issuerAuthState,
-    );
-    if (!issuerStateResolved) {
-      throw new Error(`issuer state is not valid`);
```

Comment:
where is this error now? I can't find it, but we have it in golang version

Reply:
this error will never be thrown according to current implementation